test/bdd/steps/network.js

```diff
@@ -755,3 +755,27 @@ Given(/^(\d+)[st|nd|rd|th]+ bootstrap should reply on info route$/, { timeout: 3
         'network', 'is_bootstrap',
     ]);
 });
+
+Given(/^selected DHes should be payed out after holding times is up*$/, { timeout: 180000 }, async function () {
+    expect(this.state.nodes.length, 'No started nodes').to.be.greaterThan(0);
+
+    const myPromises = [];
+
+    // slice(1) to exlude DC node
+    this.state.nodes.slice(1).forEach((node) => {
+        myPromises.push(new Promise((accept) => {
+            // node.state.takenBids gets populated only for choosen DH nodes
+            if (node.state.takenBids.length === 1) {
+                node.once('dh-pay-out-finalized', async () => {
+                    const myBalance = await httpApiHelper.apiBalance(node.state.node_rpc_url, false);
+                    expect(Number(myBalance.profile.staked) - Number(node.options.nodeConfiguration.initial_deposit_amount)).to.be.closeTo(Number(node.options.nodeConfiguration.dc_token_amount_per_holder), 1000000);
```

Comment:
Do not use Number objects when calculating with weis. Please use big numbers (BN) instead.

Reply:
Also, those numbers has to be exact not closed to.